### PR TITLE
stat/distuv: provide complete test coverage for Pareto distribution and implement Pareto Quantile function

### PR DESCRIPTION
**Pull request by romanwerpachowski**
Also fixes a bug in Pareto ExKurtosis, simplifies the implementation of Pareto Survival and improves the accuracy of Pareto CDF for x ~= xM.

Please take a look.

### CODE REVIEW

File: stat/distuv/pareto.go [hunk 1]
```diff
@@ -88,6 +88,14 @@ func (p Pareto) Prob(x float64) float64 {
 	return math.Exp(p.LogProb(x))
 }
 
+// Quantile returns the inverse of the cumulative probability distribution.
+func (p Pareto) Quantile(prob float64) float64 {
+	if prob < 0 || prob > 1 {
```
Comment (kortschak) [inline stat/distuv/pareto.go:93]: The convention here is `prob < 0 || 1 < prob`. This makes the condition reflect the arrangement on the number line.

Comment (romanwerpachowski) [reply stat/distuv/pareto.go:93]: OK. I copied this from another file ;-)

**Comment by romanwerpachowski** [comment]
PTAL.

File: stat/distuv/pareto.go (hunk 1 above)
Comment (kortschak) [reply stat/distuv/pareto.go]: Yeah, no worries. The conventions evolve.

**Comment by kortschak** [comment]
Suggest commit message: "stat/distuv: add Pareto.Quantile and improve Pareto tests", probably also a bullet list of smaller items like the refactors/re-expressions.

**Pull request merged**